src/SonarScanner.MSBuild.TFS/CoverageReportProcessorBase.cs

```diff
@@ -122,5 +122,8 @@ private bool TryConvertCoverageReports(IEnumerable<string> vsCoverageFilePaths,
             vsCoverageXmlPaths = xmlFileNames;
             return true;
         }
+
+        private static void WriteProperty(string propertiesFilePath, string property, string[] paths) =>
```

Comment:
NIT: Because you do a replace it may be better to name the method:
```suggestion
        private static void WriteFileProperty(string propertiesFilePath, string property, string[] paths) =>
```

Reply:
It's not a replacement, we do an append. 

I would not change it. For me, the "file" word does not bring more clarity.

Reply:
The replace was regarding `\` -> `\\` The WriteProperty can not be universally used. Just for "File" properties.